Lib/concurrent/futures/process.py

```diff
@@ -591,7 +591,7 @@ def submit(self, fn, *args, **kwargs):
         with self._shutdown_lock:
             if self._broken:
                 raise BrokenProcessPool(self._broken)
-            if self._shutdown_thread:
+            if _global_shutdown or self._shutdown_thread:
```

Comment:
Maybe add an explicit `RuntimeError` stating that the submit failed because the interpreter is shutting down.

Reply:
I changed this to an explicit `RuntimeError` in 881924f5bb37a38529ed2768d0227a5c528b45f0